packages/web3/types/tests/web3-test.ts

```diff
@@ -31,7 +32,13 @@ Web3.modules;
 // $ExpectType Providers
 Web3.providers;
 
-const web3 = new Web3('https://localhost:5000/');
+let web3 = new Web3('https://localhost:5000/');
+
+const netSocket = new net.Socket();
```

Comment:
Can you move these to the bottom so they don’t effect the main tests  and then use the the expect type syntax to test it.

Like:

$ExpectType Web3
web3 = new Web3()

$ExpectType Web3
web3 = new Web3(‘https://localhost:5000/‘, netSocket)

Above the new web3 that you create. Then looks good to merge.